CMakeLists.txt

```diff
@@ -71,7 +71,7 @@ endif ()
 
 # Python Bindings
 option(FICTION_PYTHON_BINDINGS "Build fiction Python bindings" OFF)
-if (FICTION_EXPERIMENTS)
+if (FICTION_PYTHON_BINDINGS)
     message(STATUS "Building fiction experiments")
```

Comment:
This line should then probably also say "Building fiction Python bindings" instead